packages/core/src/external/fhir/patient/index.ts

```diff
@@ -16,6 +16,12 @@ export const genderMapping: { [k in GenderAtBirth]: "female" | "male" } = {
   M: "male",
 };
 
+export function genderToFhir(
+  gender: GenderAtBirth
+): "male" | "female" | "other" | "unknown" | undefined {
+  return genderMapping[gender];
+}
```

Comment:
@thomasyopes IIRC you're also introducing something for this, right? Whoever merges last to point to the other one's version, please.

Reply:
Note: conflict PR Is merged https://github.com/metriport/metriport/pull/2025